rocketpy/simulation/flight.py

```diff
@@ -3173,6 +3175,32 @@ class attributes which are instances of the Function class. Usage
             encoding="utf-8",
         )
 
+    def export_sensor_data(self, file_name, sensor=None):
```

Comment:
It would be nice to also allow receiving the sensor's name

Reply:
Receive the sensor name to choose the sensor you want to export data from? 

You mean to receive the sensor name in the `sensor` argument or create another argument called `sensor_name`?

Reply:
all the sensors objects have a "name" attribute, no?

Reply:
https://github.com/RocketPy-Team/RocketPy/pull/636/commits/9ce718eaf809dc5b3059e05f08a313388af3a89e